com.unity.render-pipelines.core/Editor/CoreEditorStyles.cs

```diff
@@ -88,6 +95,10 @@ static CoreEditorStyles()
             contextMenuIcon = new GUIContent(paneOptionsIcon, contextTooltip);
             contextMenuStyle = new GUIStyle("IconButton");
 
+            redTexture = CoreEditorUtils.CreateColoredTexture2D(Color.red);
+            greenTexture = CoreEditorUtils.CreateColoredTexture2D(Color.green);
+            blueTexture = CoreEditorUtils.CreateColoredTexture2D(Color.blue);
```

Comment:
IIRC, Texture2D assets are not garbage collected, they need to be explicitly destroyed otherwise they'll leak in the editor.

Reply:
Added `DestroyInmediate` of this textures.